tests/rpmbuild.at

```diff
@@ -2272,3 +2272,46 @@ runroot rpmbuild \
 ],
 [ignore])
 AT_CLEANUP
+
+# ------------------------------
+# Check if dynamic spec generation works
+AT_SETUP([rpmbuild with dynamic spec generation])
+AT_KEYWORDS([build])
+RPMDB_INIT
+AT_CHECK([
+
+runroot rpmbuild -ba /data/SPECS/dynamic.spec
+],
+[0],
+[ignore],
+[ignore])
+
+AT_CHECK([
+
+runroot rpm -qp --qf "%{Summary}\n" /build/RPMS/noarch/dynamic-docs-1.0-1.noarch.rpm
+runroot rpm -ql /build/RPMS/noarch/dynamic-docs-1.0-1.noarch.rpm
+],
+[0],
+[Documentation for dynamic spec
+/usr/local/share/doc/dynamic-docs-1.0
+/usr/local/share/doc/dynamic-docs-1.0/FAQ
```

Comment:
This will fail when rpm is built with prefix other than /usr/local, such as distros would normally do.
Run the above rpmbuild with `--define "_prefix /usr/local"` to fix.

Reply:
Done.